autoload/gina/command/blame.vim

```diff
@@ -265,7 +265,7 @@ endfunction
 
 function! s:is_sign_visible() abort
   if !exists('&signcolumn') || &signcolumn ==# 'auto'
-    return len(split(execute('sign place buffer=0'), '\r\?\n')) > 1
+    return len(split(execute('sign place buffer='.bufnr('%')), '\r\?\n')) > 1
```

Comment:
Please add spaces around `.` while Vim use `.` for 1) concatenating strings 2) dictionary attribute. Without spaces, it is a bit difficult to distinguish